### PR TITLE
vala 0.29.1

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

File: Library/Formula/vala.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
 class Vala < Formula
   desc "Compiler for the GObject type system"
   homepage "https://live.gnome.org/Vala"
-  url "http://ftp.gnome.org/pub/GNOME/sources/vala/0.26/vala-0.26.1.tar.xz"
-  sha256 "8407abb19ab3a58bbfc0d288abb47666ef81f76d0540258c03965e7545f59e6b"
+  url "http://ftp.gnome.org/pub/GNOME/sources/vala/0.29/vala-0.29.1.tar.xz"
```
Comment (bfontaine) [inline Library/Formula/vala.rb]: This is an odd version number, meaning it’s a [development release](http://bot.brew.sh/job/Homebrew%20Pull%20Requests/27909/version=mavericks/testReport/junit/brew-test-bot/mavericks/audit_vala/).

Comment (nijikon) [inline Library/Formula/vala.rb]: @bfontaine no problem, will bring back the devel section and update it to 0.29.1, is 0.26.2 considered stable?

Comment (bfontaine) [inline Library/Formula/vala.rb]: Even versions numbers are stable, odd ones are devel, so yes. You might want to check if there’s a 0.28.x release.

**Comment by nijikon** [comment]
@bfontaine done

**Comment by mikemcquaid** [comment]
Thanks @nijikon!

**Pull request closed without merging**